### PR TITLE
chore: return user uuid in open id identities

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: 

### Description:

Update open id identities queries to return the user uuid instead of the user id.
Unblocks work for #9692  

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/generated/routes.ts [hunk 1]
```diff
@@ -5778,7 +5778,7 @@ const models: TsoaRoute.Models = {
     // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
     OpenIdIdentityIssuerType: {
         dataType: 'refEnum',
-        enums: ['google', 'okta', 'oneLogin', 'azuread'],
+        enums: ['google', 'okta', 'oneLogin', 'azuread', 'oidc'],
```
Comment (ZeRego) [inline packages/backend/src/generated/routes.ts:5781]: Unrelated change from running `yarn generate-api`

**Comment by owlas** [comment]
@ZeRego could you explain briefly why you need this?

**Comment by ZeRego** [comment]
> @ZeRego could you explain briefly why you need this?

@owlas To get the user session via user uuid in the following [PR](https://github.com/lightdash/lightdash/pull/9706/files#diff-79e0757ba90eb4bb1ee66d5145ae5b0e4a647facdc6bd5d7192e44ef1c609200R538).

**Review by owlas — APPROVED** [review]
